Update troubleshooting.md (#7099)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/troubleshooting.md

Analytics.js generates a campaign object inside the context object whenever the URL contains search parameters. Without any UTM parameters, the campaign object remains empty. 

> ## Why Are Events Received with Timestamps Set in the Past or Future?

Reviewer: ```suggestion
## Why do I see events with timestamps in the past or future?
```

> Please see full details [here](https://segment.com/docs/connections/spec/common/#why-are-events-received-with-timestamps-set-in-the-past-or-future).

Reviewer: ```suggestion
You may see events with timestamp discrepancies due to manual overriding of the timestamp value, mobile apps closed or set in the background, traffic from bots, or inaccurate device or browser time. For more information, see Segment's [Common Fields Spec](/docs/connections/spec/common/#why-are-events-received-with-timestamps-set-in-the-past-or-future).
```
